packages/x-date-pickers/src/locales/esES.ts

```diff
@@ -74,7 +74,7 @@ const esESPickers: Partial<PickersLocaleText<any>> = {
   dateTableLabel: 'elige la hora',
 
   // Field section placeholders
-  fieldYearPlaceholder: (params) => 'Y'.repeat(params.digitAmount),
+  fieldYearPlaceholder: (params) => 'A'.repeat(params.digitAmount),
```

Comment:
Is it really common to use such a format (placeholder) in Spain for dates?
This would make `DD/MM/YYYY` become `DD/MM/AAAA`.
Based on a quick search, I was not able to find resources confirming if such an approach is common.
Could you confirm if this is really expected? 🤔 

https://en.wikipedia.org/wiki/Date_and_time_notation_in_Spain

Reply:
Of course, see the page that explains date (fecha) on Wikipedia in Spanish. All references to the year are 'AAAA'

https://es.wikipedia.org/wiki/Fecha

Reply:
@eduardodallmann Thank you for the confirmation! 🙏